docker-compose.yml

```diff
@@ -17,8 +17,8 @@ services:
       timeout: 20s
       retries: 10
 
-  mailhog:
-    container_name: mailhog
+  smtp:
+    container_name: veryrezsi_smtp
```

Comment:
If the container name is no longer mailhog, then the docker config files should reflect that, as the DNS host resolve is based upon this.